Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM centos:7.2.1511
+
+RUN yum -y install  gcc \
```

Comment:
This configuration is indeed CentOS specific. By calling `FROM centos:7.2.1511` at line 1, docker is retrieving the [CentOS official](https://hub.docker.com/_/centos/) 7.2.1511 image to run the subsequent commands through. The FROM command documentation can be found [here](https://docs.docker.com/engine/reference/builder/#/from).